### ISSUE TITLE
INI parser: Remove single quotes from values

### ISSUE THREAD

**Issue by kellyjonbrazil** [kellyjonbrazil/jc#254]
If you have a `~/.my.cnf` file like this:

```
[client]
user=foo
host=localhost
password='bar'
```

Then JC (via the Ansible filter and the KV parser in this case) returns:

```json
        "client": {
            "host": "localhost",
            "password": "'bar'",
            "user": "foo"
        }
```

And then attempts to login using this password fail due to the extra single quotes.

An answer to this issue is to replace the single quotes with double quotes in the `~/.my.cnf` file.

_Originally posted by @chriscroome in https://github.com/kellyjonbrazil/jc/issues/253#issuecomment-1150145129_

**Comment by kellyjonbrazil** [comment]
I have this added in `dev`: https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/ini.py

This will go into the next release.

**Comment by kellyjonbrazil** [comment]
This is now available in v1.20.1 available on PIP. (binaries and packages will be available soon)
